src/controllers/notificationsettings.js

```diff
@@ -1,60 +1,30 @@
-                    if (showHelp) {
-                        showHelp = false;
-                        itemHtml += '<a is="emby-linkbutton" class="raised button-alt headerHelpButton" target="_blank" href="https://web.archive.org/web/20181216120305/https://github.com/MediaBrowser/Wiki/wiki/Notifications">';
-                        itemHtml += globalize.translate("Help");
-                        itemHtml += "</a>";
-                    }
-                    itemHtml += "</div>";
-                    itemHtml += '<div class="paperList">';
-                }
-                itemHtml += '<a class="listItem listItem-border" is="emby-linkbutton" data-ripple="false" href="notificationsetting.html?type=' + notification.Type + '">';
-                if (notification.Enabled) {
-                    itemHtml += '<i class="listItemIcon md-icon">notifications_active</i>';
-                } else {
-                    itemHtml += '<i class="listItemIcon md-icon" style="background-color:#999;">notifications_off</i>';
-                }
-                itemHtml += '<div class="listItemBody">';
-                itemHtml += '<div class="listItemBodyText">' + notification.Name + "</div>";
-                itemHtml += "</div>";
-                itemHtml += '<button type="button" is="paper-icon-button-light"><i class="md-icon">mode_edit</i></button>';
-                itemHtml += "</a>";
-                return itemHtml;
-            }).join("");
-
-            if (list.length) {
-                html += "</div>";
-                html += "</div>";
-            }
-            page.querySelector(".notificationList").innerHTML = html;
-            loading.hide();
+                return i.Category != lastCategory && (lastCategory = i.Category, lastCategory && (itemHtml += "</div>", itemHtml += "</div>"), itemHtml += '<div class="verticalSection verticalSection-extrabottompadding">', itemHtml += '<div class="sectionTitleContainer" style="margin-bottom:1em;">', itemHtml += '<h2 class="sectionTitle">', itemHtml += i.Category, itemHtml += "</h2>", showHelp && (showHelp = !1, itemHtml += '<a is="emby-linkbutton" class="raised button-alt headerHelpButton" target="_blank" href="https://web.archive.org/web/20181216120305/https://github.com/MediaBrowser/Wiki/wiki/Notifications">' + globalize.translate("Help") + "</a>"), itemHtml += "</div>", itemHtml += '<div class="paperList">'), itemHtml += '<a class="listItem listItem-border" is="emby-linkbutton" data-ripple="false" href="notificationsetting.html?type=' + i.Type + '">', i.Enabled ? itemHtml += '<i class="listItemIcon md-icon">notifications_active</i>' : itemHtml += '<i class="listItemIcon md-icon" style="background-color:#999;">notifications_off</i>', itemHtml += '<div class="listItemBody">', itemHtml += '<div class="listItemBodyText">' + i.Name + "</div>", itemHtml += "</div>", itemHtml += '<button type="button" is="paper-icon-button-light"><i class="md-icon">mode_edit</i></button>', itemHtml += "</a>"
```

Comment:
this also looks reversed accidentally

Reply:
i reverted to master state and also replaced emby-linkbutton with emby-button.